src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/CreateCohortWithDraftApprenticeshipController.cs

```diff
@@ -82,20 +87,29 @@ public async Task<IActionResult> AddDraftApprenticeship(AddDraftApprenticeshipVi
             catch (CommitmentsApiModelException ex)
             {
                 ModelState.AddModelExceptionErrors(ex);
-                await AddLegalEntityAndCoursesToModel(model);
+                await AddCoursesAndProviderNameToModel(model);
                 return View(model);
             }
         }
 
-        private async Task AddLegalEntityAndCoursesToModel(DraftApprenticeshipViewModel model)
+        private async Task AddCoursesAndProviderNameToModel(DraftApprenticeshipViewModel model)
         {
             var courses = await GetCourses();
             model.Courses = courses;
+
+            var providerName = await GetProviderName(model.ProviderId);
+            model.ProviderName = providerName;
         }
 
         private Task<IReadOnlyList<ITrainingProgramme>> GetCourses()
         {
             return _trainingProgrammeApiClient.GetAllTrainingProgrammes();
         }
+
+        private async Task<string> GetProviderName(long providerId)
+        {
+            var blah = await _commitmentsApiClient.GetProvider(providerId);
```

Comment:
noooop! rename plz.

Reply:
This change appears to have come as a result of a merge fail when pulling master into CV-192 resulting in commit 068e4c3b74313e348a419a7d0add1cce12f1515e being skipped. I've hard reset the branch, pulled from master and force pushed, which solved the issue.